### DISCUSSION TITLE
When trying to dowload, Downloadhelper think that it's already downloaded and also do not download in the required resolution

### DISCUSSION THREAD

**Discussion by DevRGT** [aclap-dev/video-downloadhelper#339] [Help and Q&A]
### Your question

Hello,

During the COVID period, in France they were school through TV. At the begining the video could be replay. Today it's more complicated.

When I try to download some videos of this list:
https://www.lumni.fr/article/la-maison-lumni-le-programme-pour-les-eleves-de-primaire#CM2
or better here:
https://www.lumni.fr/serie/la-maison-lumni-primaire#5

<img width="559" alt="Capture d’écran 2024-03-28 à 10 00 43" src="https://github.com/aclap-dev/video-downloadhelper/assets/40140101/eda7a06f-ede4-4a01-bc77-aa45c258bf02">

There are for exemple several videos named:
Activités géométriques et programmation - 1/6
Activités géométriques et programmation - 2/6
Activités géométriques et programmation - 3/6
Activités géométriques et programmation - 4/6
Activités géométriques et programmation - 5/6
Activités géométriques et programmation - 6/6

When I want to dowload the next videos Downloadhelper think that is the same that the vido downloaded just before.

Also the resolution is not the one chosen and I get a lower resolution 

and last issue, I have some errors:
<img width="499" alt="Capture d’écran 2024-03-28 à 10 13 00" src="https://github.com/aclap-dev/video-downloadhelper/assets/40140101/f7db808a-b4b2-4889-8254-6d7a9d9e361e">

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: fr
license: unset
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:124.0) Gecko/20100101 Firefox/124.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/Users/macbookpro13-16-512",
  "converterBinary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "n6.1-11-gc30f360955"
}

**Comment by Mattmjs** [comment]
Have you got an example that gives the exit code error. I don't know about the other issues.
